.gitignore

```diff
@@ -15,3 +15,5 @@ npm-debug.log
 
 # Regularly changing or system specific IDE files
 /.idea/workspace.xml
+build/
```

Comment:
Committing the build repo is required since released are served via Github Pages directly to this URL.